
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Principal(object):
    def setupUi(self, Principal):
        Principal.setObjectName("Principal")
        Principal.resize(965, 708)
        Principal.setMinimumSize(QtCore.QSize(965, 708))
        Principal.setMaximumSize(QtCore.QSize(965, 708))
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("../primecondoDesk/img/logo.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        Principal.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(Principal)
        self.centralwidget.setObjectName("centralwidget")
        Principal.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(Principal)
        self.statusbar.setObjectName("statusbar")
        Principal.setStatusBar(self.statusbar)
        self.menubar = QtWidgets.QMenuBar(Principal)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 965, 21))
        self.menubar.setObjectName("menubar")
        self.menuFile = QtWidgets.QMenu(self.menubar)
        self.menuFile.setObjectName("menuFile")
        self.menuEdit = QtWidgets.QMenu(self.menubar)
        self.menuEdit.setObjectName("menuEdit")
        self.menuCadastro_de_Visitantes = QtWidgets.QMenu(self.menubar)
        self.menuCadastro_de_Visitantes.setObjectName("menuCadastro_de_Visitantes")
        Principal.setMenuBar(self.menubar)
        self.actionCadastro = QtWidgets.QAction(Principal)
        self.actionCadastro.setObjectName("actionCadastro")
        self.actionSenha = QtWidgets.QAction(Principal)
        self.actionSenha.setObjectName("actionSenha")
        self.btnActAtualizarMoradores = QtWidgets.QAction(Principal)
        self.btnActAtualizarMoradores.setObjectName("btnActAtualizarMoradores")
        self.btnActAtivarDesativarMoradores = QtWidgets.QAction(Principal)
        self.btnActAtivarDesativarMoradores.setObjectName("btnActAtivarDesativarMoradores")
        self.btnActNovoVisitante = QtWidgets.QAction(Principal)
        self.btnActNovoVisitante.setObjectName("btnActNovoVisitante")
        self.actionDependente = QtWidgets.QAction(Principal)
        self.actionDependente.setObjectName("actionDependente")
        self.actionAutonomo = QtWidgets.QAction(Principal)
        font = QtGui.QFont()
        font.setPointSize(8)
        self.actionAutonomo.setFont(font)
        self.actionAutonomo.setObjectName("actionAutonomo")
        self.actionDependente_2 = QtWidgets.QAction(Principal)
        self.actionDependente_2.setObjectName("actionDependente_2")
        self.actionPropriet_rio = QtWidgets.QAction(Principal)
        self.actionPropriet_rio.setObjectName("actionPropriet_rio")
        self.actionDependente_3 = QtWidgets.QAction(Principal)
        self.actionDependente_3.setObjectName("actionDependente_3")
        self.actionSair = QtWidgets.QAction(Principal)
        self.actionSair.setObjectName("actionSair")
        self.actionCadastro_de_Visitantes = QtWidgets.QAction(Principal)
        self.actionCadastro_de_Visitantes.setObjectName("actionCadastro_de_Visitantes")
        self.btnActConsultarVisitantes = QtWidgets.QAction(Principal)
        self.btnActConsultarVisitantes.setObjectName("btnActConsultarVisitantes")
        self.actionAtualizarVisitantes = QtWidgets.QAction(Principal)
        self.actionAtualizarVisitantes.setObjectName("actionAtualizarVisitantes")
        self.actionAtivar_Visitantes = QtWidgets.QAction(Principal)
        self.actionAtivar_Visitantes.setObjectName("actionAtivar_Visitantes")
        self.btnActAtivarDesativarVisitantes = QtWidgets.QAction(Principal)
        self.btnActAtivarDesativarVisitantes.setObjectName("btnActAtivarDesativarVisitantes")
        self.actionCadastrarPrestador = QtWidgets.QAction(Principal)
        self.actionCadastrarPrestador.setObjectName("actionCadastrarPrestador")
        self.actionCadastrarFuncionario = QtWidgets.QAction(Principal)
        self.actionCadastrarFuncionario.setObjectName("actionCadastrarFuncionario")
        self.actionPropriet_rio_2 = QtWidgets.QAction(Principal)
        self.actionPropriet_rio_2.setObjectName("actionPropriet_rio_2")
        self.actionMorador = QtWidgets.QAction(Principal)
        self.actionMorador.setObjectName("actionMorador")
        self.btnActCadastrarMoradores = QtWidgets.QAction(Principal)
        self.btnActCadastrarMoradores.setObjectName("btnActCadastrarMoradores")
        self.btnActConsultarMoradores = QtWidgets.QAction(Principal)
        self.btnActConsultarMoradores.setObjectName("btnActConsultarMoradores")
        self.menuFile.addSeparator()
        self.menuFile.addAction(self.actionCadastro)
        self.menuFile.addAction(self.actionSenha)
        self.menuFile.addAction(self.actionSair)
        self.menuEdit.addAction(self.btnActCadastrarMoradores)
        self.menuEdit.addAction(self.btnActConsultarMoradores)
        self.menuEdit.addAction(self.btnActAtualizarMoradores)
        self.menuEdit.addAction(self.btnActAtivarDesativarMoradores)
        self.menuCadastro_de_Visitantes.addAction(self.btnActNovoVisitante)
        self.menuCadastro_de_Visitantes.addAction(self.btnActConsultarVisitantes)
        self.menuCadastro_de_Visitantes.addAction(self.btnActAtivarDesativarVisitantes)
        self.menubar.addAction(self.menuFile.menuAction())
        self.menubar.addAction(self.menuEdit.menuAction())
        self.menubar.addAction(self.menuCadastro_de_Visitantes.menuAction())

        self.retranslateUi(Principal)
        QtCore.QMetaObject.connectSlotsByName(Principal)

    def retranslateUi(self, Principal):
        _translate = QtCore.QCoreApplication.translate
        Principal.setWindowTitle(_translate("Principal", "Prime Condo"))
        self.menuFile.setTitle(_translate("Principal", "CONTA"))
        self.menuEdit.setTitle(_translate("Principal", "CONDÔMINOS"))
        self.menuCadastro_de_Visitantes.setTitle(_translate("Principal", "VISITANTES"))
        self.actionCadastro.setText(_translate("Principal", "Cadastro"))
        self.actionSenha.setText(_translate("Principal", "Senha"))
        self.btnActAtualizarMoradores.setText(_translate("Principal", "Atualizar Moradores"))
        self.btnActAtivarDesativarMoradores.setText(_translate("Principal", "Ativar / Desativar - Moradores"))
        self.btnActNovoVisitante.setText(_translate("Principal", "Novo Visitante"))
        self.actionDependente.setText(_translate("Principal", "Dependente"))
        self.actionAutonomo.setText(_translate("Principal", "Autonômo"))
        self.actionDependente_2.setText(_translate("Principal", "Proprietário"))
        self.actionPropriet_rio.setText(_translate("Principal", "Morador"))
        self.actionDependente_3.setText(_translate("Principal", "Dependente"))
        self.actionSair.setText(_translate("Principal", "Sair"))
        self.actionCadastro_de_Visitantes.setText(_translate("Principal", "Cadastro de Visitantes"))
        self.btnActConsultarVisitantes.setText(_translate("Principal", "Consulta de Visitantes"))
        self.actionAtualizarVisitantes.setText(_translate("Principal", "Atualizar Visitantes"))
        self.actionAtivar_Visitantes.setText(_translate("Principal", "Ativar Visitantes"))
        self.btnActAtivarDesativarVisitantes.setText(_translate("Principal", "Desativar / Ativar - Visitantes"))
        self.actionCadastrarPrestador.setText(_translate("Principal", "Cadastrar Prestador "))
        self.actionCadastrarFuncionario.setText(_translate("Principal", "Cadastrar Funcionário"))
        self.actionPropriet_rio_2.setText(_translate("Principal", "Proprietário"))
        self.actionMorador.setText(_translate("Principal", "Morador"))
        self.btnActCadastrarMoradores.setText(_translate("Principal", "Cadastro de Moradores"))
        self.btnActConsultarMoradores.setText(_translate("Principal", "Consulta de Moradores"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Principal = QtWidgets.QMainWindow()
    ui = Ui_Principal()
    ui.setupUi(Principal)
    Principal.show()
    sys.exit(app.exec_())
